code-challenges.test.js

```diff
@@ -40,17 +63,32 @@ const studyMinutesWeek1 = {
 }
 // Expected output: [15, 15, 20, 30, 30, 60, 90]
 
-const studyMinutesWeek2 = {
-  sunday: 100,
-  monday: 10,
-  tuesday: 45,
-  wednesday: 60,
-  thursday: 20,
-  friday: 15,
-  saturday: 65
+  const studyMinutesWeek2 = {
+    sunday: 100,
+    monday: 10,
+    tuesday: 45,
+    wednesday: 60,
+    thursday: 20,
+    friday: 15,
+    saturday: 65
 }
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
 
 // b) Create the function that makes the test pass.
-
+const objectValueSorted = (obj) => {
+  // get an array of the values from the objects
+  const valuesArray = Object.values(obj) 
+  // create a copy using .slice() and sort it in ascending order
+  const valuesSorted = valuesArray.slice().sort((value1, value2) => {
+    // compare value 1 & value 2
+    if(value1 < value2){
+      return -1
+    } else if(value1 > value2){
+      return 1
+    } else {
+      return 0
+    }
+  })
+  return valuesSorted
+}
```

Comment:
Wow Danny!  You did a lot of work here!  You broke down the sort method, which is AWESOME!  I encourage you to look up the .sort() method and refactor this code.  It would be a good learning experience. If you can write this code above, you are doing just fine!